src/util.rs

```diff
@@ -128,3 +132,35 @@ pub fn output_report(report: &CrashReport, matches: &ArgMatches, argv: &[&str])
     }
     Ok(())
 }
+
+pub fn change_ignored_frames(path: &Path) -> error::Result<()> {
```

Comment:
Maybe we should move it to stacktrace_constants.rs?

Reply:
stacktrace_constants.rs now contains only constants and variables. Maybe we shouldn't move the function?